scripts/data_prep/convert_delta_to_json.py

```diff
@@ -368,36 +369,52 @@ def fetch(
         dbsql (databricks.sql.connect): dbsql session
     """
     cursor = dbsql.cursor() if dbsql is not None else None
+    for row_retry in range(MAX_RETRY):
```

Comment:
Can we reuse Composer's [retry logic] here (https://github.com/mosaicml/composer/blob/dev/composer/utils/retrying.py)? It has better backoff and jitter etc. (you may need to refactor the functions here a bit to use the decorator).